testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestHiveAndDeltaLakeCompatibility.java

```diff
@@ -53,4 +54,18 @@ public void testInformationSchemaColumnsOnPresenceOfHiveView()
             onTrino().executeQuery("DROP SCHEMA " + schemaName);
         }
     }
+
```

Comment:
Let's maybe add a test where we create a table, we unregister it via trino and we verify in both Trino & Delta OSS/Databricks that the table is not anymore visible.

Reply:
Added another test case.